htdocs/js/jquery.dynamictable.js

```diff
@@ -1,4 +1,29 @@
 /*global window, jQuery*/
+
+/**
+ * This is a wrapper for a scrollable table that dynamically
+ * changes depending on weather or not the table has overflow
+ * or not. The table can be setup to freeze a column once the
+ * column comes to the left of the screen. The column will
+ * unfreeze once the column to its right is fully visiable.
+ *
+ * To setup the table without freezing a column simply call
+ * the function DynamicTable on the table you want to apply
+ * the scrolling using the standard jquery function calling.
+ * For this the fuction does not take in any parameters.
+ *
+ * To setup the table with the freezing column, call in a
+ * similar manner to the way without the freezing column.
+ * The only diffrence being you pass an object of the structure
+ *
+ * { "freezeColumn" : "columnID" }
+ *
```

Comment:
This is a pretty good comment, but it would be clearer with an example:

ie:
$("#cand").DynamicTable({ "freezeColumn" : "pscid" });

(I just copy/pasted it from where it's used below)